src/2geom/convex-hull.h

```diff
@@ -203,7 +203,7 @@ class ConvexHull {
     /// @name Iterate over points.
     /// @{
     /** @brief Get the begin iterator to the points that form the hull.
-     * Points are are returned beginning the the leftmost one, going along
+     * Points are returned from the beginning of the leftmost one, going along
```

Comment:
Should be: Points are returned beginning with the leftmost one

Reply:
fixed. repushed PR